application.yml

```diff
@@ -54,7 +54,10 @@ management:
     export:
       prometheus:
         enabled: true
-
+  management:
```

Comment:
Are you sure about the indentation?
Moreover there already is a `management` property.